### ISSUE TITLE
Mac mouse fix wont start anymore after accidental start from recycle bin [bug]

### ISSUE THREAD

**Issue by fabianboerner** [noah-nuebling/Mac-Mouse-Fix#844]
## Description

I accidently started mac mouse fix from recycle bin now i cant enable it if the app is not in the recycle bin.

Deleting it didnt help.

## Steps to reproduce

1. Download mac mosue fix
2. put into recycle bin
3. start from recycle bin
4. enable
5. disable and purge from recycle bin
6. copy to programms 
7. start mac mouse fix -> will not activate

starting from the short cut will tell you it cant start the programm because its in the recycle bin

## Context

Mac Mouse Fix Version: 3.0.1
macOS Version: 14.2.1
Mouse Model: –

## Additional Comments

–

---

### Console Logs

–

### Crash Reports

–

**Comment by gkrawiec** [comment]
I have the same problem, but not with the trash. I downloaded the app and started it from the downloads folder.
Then i moved it to apps.
Then I turned mac mouse fix and it wont activate unless I move the app back into the downloads folder.

I had to use Uninstaller to get rid of mac mouse fix and all its components, and then resinstall for it to work.

**Comment by fabianboerner** [comment]
Yes i did the same but as it was in recycle bin it was a bit more complicated of useing a deinstaller then moving it back into recycle bin from application.

**Comment by noah-nuebling** [comment]
**Update**
I now wrote a [guide](https://github.com/noah-nuebling/mac-mouse-fix/discussions/861) for solving this issue to replace the solution steps in [this comment](https://github.com/noah-nuebling/mac-mouse-fix/issues/648#issuecomment-1872560379) (which I talked about below). I hope it works well for everyone. If you have any feedback, let me know.

---

Hey guys, try the steps outlined in [this comment](https://github.com/noah-nuebling/mac-mouse-fix/issues/648#issuecomment-1872560379) that should fix these kinds of issues. 

If not please let me know. 

I'm trying to address this as best as possible.
